### PR TITLE
Routing for the main table and the addition of the maps table

### LINKED ISSUE #7: research springboot and mySQL

### LINKED ISSUE #32: create server routing for the location component

### PR DESCRIPTION
**Pull request by NeelyAnne**
closes #32 
closes #7

### CODE REVIEW

**Review by mrterranova — APPROVED** [review]
Looks good. Thank you Neely

File: EarthBnB/src/main/java/com/Earthbb/Map/MapController.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+public class MapController {
+
+    @Autowired
+    private MapRepository mapRepository;
+
+    @GetMapping
+    public List<Map> list() { return mapRepository.findAll();}
+
+    @GetMapping("{id}")
+    public Map get(@PathVariable("id") long id) { return mapRepository.getOne(id);}
```
Comment (Trevis42) [inline EarthBnB/src/main/java/com/Earthbb/Map/MapController.java:23]: does this need "/{id}" or is "{id}" fine? cause your request mapping doesnt have and '/' on the end, so if you use GET localhost:8080/api/v1/map/# ... would that fail or does it put on on the end when you do that so it doesn matter?

Comment (Trevis42) [reply EarthBnB/src/main/java/com/Earthbb/Map/MapController.java:23]: Esp cause delete and put have it as "/{id}" so shouldnt get as well?

**Review by Trevis42 — APPROVED** [review]
i think its all good.. just check the comments.

**Pull request merged**